### ISSUE TITLE
Style/Documentation misses Rails models

### ISSUE THREAD

**Issue by zverok** [rubocop/rubocop#9595]
I just noticed in our project there is a model, say, looking like this:

```ruby
class Something < ApplicationRecord
  belongs_to :belongs_to_something_else

  validates :stuff
end
```
And `Style/Documentation` is perfectly OK with it being undocumented (due to statement [here](https://github.com/rubocop/rubocop/blob/master/lib/rubocop/cop/style/documentation.rb#L96), I assume): 
```
return if macro_only?(body)
```

I'd argue that this model **should** be documented: it is not an "empty class, still in development, don't bother the developer", but actually production-ready AR model... lacking any docs.
Even more than that, I'd argue that even this:
```ruby
class Something < ApplicationRecord
end
```
...requires docs. Because this class is actually very non-empty (can have a lot of behavior, due to inheritance from AR, and **needs** an explanation "model of what this is").

So, the takeout of this is: 
* probably `Style/Documentation` should include "macro-only classes" by default? (For me, actually, it is hard to imagine a situation where the opposite is more useful; but I might just miss something very under my nose)
* probably `Style/Documentation` needs configuration whether it allows empty classes (then, it can be fine-tuned by user to "generall allow, but disallow in `app/models`")

WDYT?

**Comment by bbatsov** [comment]
I agree with both points. We should make this "hidden" behaviour a configuration option and I'd personally set it to `false`, although this would be kind of a breaking change.
